package.json

```diff
@@ -10,17 +10,28 @@
   "license": "Apache 2.0",
   "unity": "2019.4",
   "unityRelease": "0f1",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "dependencies": {
     "com.unity.nuget.newtonsoft-json": "3.0.2",
-    "voxell.util": "1.4.2",
-    "com.github.asus4.tflite": "2.4.1"
+    "com.github.asus4.tflite": "2.4.1",
+	"com.cysharp.unitask": "2.2.5"
```

Comment:
I think the indentation is off, no big deal. I'm gonna change 2 spaces to 4 spaces later on anyway to match the C# standard. Changing back to 2 spaces will help when doing the conversion (the conversion rely on consistency)